compute/api/Compute.Samples.Tests/ComputeFixture.cs

```diff
@@ -54,7 +54,14 @@ public class ComputeFixture : IDisposable, ICollectionFixture<ComputeFixture>
 
         private StorageClient StorageClient { get; } = StorageClient.Create();
 
-        public RetryRobot Assert { get; } = new RetryRobot { RetryWhenExceptions = new Type[] { typeof(XunitException) } };
+        public RetryRobot AssertConcurently { get; } = new RetryRobot
+        { 
+            RetryWhenExceptions = new Type[] { typeof(XunitException) },
+            // We wait a little longer than default to let the concurrent test execution
+            // to finish before reattempting.
```

Comment:
Nit: remove "to" from the start of this line (the "to" in the previous line is fine).

Reply:
Done